slyd/app/components/annotations-plugin/component.js

```diff
@@ -132,10 +132,10 @@ export default Ember.Component.extend({
         },
 
         ignoreBeneath: function(_, value, index) {
-            var ignore = this.get('ignores').get(index),
+            var ignore = this.get('ignores').objectAt(index),
                 ignoreData = this.get('alldata').findBy('tagid', ignore.tagid);
-            ignore.ignoreBeneath = value;
-            ignoreData.ignoreBeneath = value;
+            ignore.set('ignoreBeneath', value);
+            ignoreData['ignore_beneath'] = value;
```

Comment:
You're right. I didn't realise that it was `ignoreData` not `ignore`.

There are some workarounds there to support an old data format. I'll be cleaning that up a bit for Ember 1.12 or 1.13 depending on the changes that they have planned.